src/dappsExplorer/DappCard.tsx

```diff
@@ -29,31 +28,18 @@ interface Props extends DappCardContentProps {
 // Since this icon exists within a touchable, make the hitslop bigger than usual
 const favoriteIconHitslop = { top: 20, right: 20, bottom: 20, left: 20 }
 
-export function DappCardContent({ dapp, onFavoriteDapp }: DappCardContentProps) {
+export function DappCardContent({ dapp, onFavoriteDapp, onUnfavoriteDapp }: DappCardContentProps) {
   const dispatch = useDispatch()
   const dappFavoritesEnabled = useSelector(dappFavoritesEnabledSelector)
   const favoriteDappIds = useSelector(favoriteDappIdsSelector)
 
   const isFavorited = favoriteDappIds.includes(dapp.id)
 
   const onPressFavorite = () => {
-    const eventProperties = isDappV2(dapp)
-      ? {
-          categories: dapp.categories,
-          dappId: dapp.id,
-          dappName: dapp.name,
-        }
-      : {
-          categoryId: dapp.categoryId,
-          dappId: dapp.id,
-          dappName: dapp.name,
-        }
-
     if (isFavorited) {
-      ValoraAnalytics.track(DappExplorerEvents.dapp_unfavorite, eventProperties)
       dispatch(unfavoriteDapp({ dappId: dapp.id }))
+      onUnfavoriteDapp?.(dapp)
     } else {
-      ValoraAnalytics.track(DappExplorerEvents.dapp_favorite, eventProperties)
```

Comment:
I'm wondering if we could pass the `section` (or other needed analytics props) so we can keep this in a single place?

Reply:
yeah this is a much better idea